.github/mergeable.yml

```diff
@@ -8,21 +8,21 @@ mergeable:
           regex: ^\[WIP\]
           message: This is work in progress. Do not merge yet.
         must_include:
-          regex: ^(feat|docs|chore|fix|test)(\(\w+\))?:.+$
+          regex: ^(feat|docs|chore|fix|test)(\(\w+\))?(:|\().+$
           message: Semantic release conventions must be followed.
       - do: description
         must_exclude:
           regex: \[ \]
           message: There are incomplete TODO task(s) unchecked.
       - do: approvals
-        min:
-          count: 2
-        or: 
+        or:
```

Comment:
Do we still need it ?

Reply:
@JKMarkowski Sorry I wasn't in the office yet. That `or` allows any of the listed reviewers _or_ minimum 2 if they are not on that list. It's up to you guys however you want to do it :)